### ISSUE TITLE
Replace pdfkit with weasyprint

### ISSUE THREAD

**Issue by chrisl8** [polo2ro/imapbox#44]
I was able to swap out pdfkit with weasyprint pretty easily and everything still works, except with fewer errors and hopefully using non-abandoned software.

If this tool is still being maintained I think it would be cool to do that "officially".

I can make a PR with my changes when I get some time if you like.

**Comment by polo2ro** [comment]
good idea, i was wondering if pdfkit was used anymore, dont hesitate to submit a PR if you want  to persist your change in the project and if this improve conversion quality, i do not use pdfkit anyway. Just be sure to make it optional for users like me
